app/javascript/components/FacetControl.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import FiltersBox from './FiltersBox';
+
+/**
+ * Converts string value to lowercase, hyphen-delimited version
+ * e.g. "Cell type" -> "cell-type"
+ */
+function slug(value) {
+  return value.toLowerCase().replace(/ /g, '-');
+}
+
+export default function FacetControl(props) {
+  
+  const [showFilters, setShowFilters] = useState(false);
+  
+  const facetName = props.facet.name;
+  const facetID = `facet-${slug(facetName)}`;
+
+  function handleClick() {
+    setShowFilters(!showFilters);
```

Comment:
What is this?

Reply:
It's code from #378.

This PR basically merges code from that PR #378 into `development` instead of `launch-advanced-search`.  Then we'll delete `launch-advanced-search` and use `development` -- for more streamlined merging.